anchore_manager/cli/utils.py

```diff
@@ -1,184 +0,0 @@
-        if 'ANCHORE_CONFIG_DIR' in settings:
-            ret['configdir'] = settings['ANCHORE_CONFIG_DIR']
-        
-    except Exception as err:
-        raise err
-
-    return(ret)
-
-def format_error_output(config, op, params, payload):
-
-    try:
-        errdata = json.loads(str(payload))
-    except:
-        errdata = {'message': str(payload)}
-
-    if config['jsonmode']:
-        ret = json.dumps(errdata, indent=4, sort_keys=True)
-        return(ret)
-
-    obuf = ""
-    try:
-        outdict = OrderedDict()    
-        if 'message' in errdata:
-            outdict['Error'] = str(errdata['message'])
-        if 'httpcode' in errdata:
-            outdict['HTTP Code'] = str(errdata['httpcode'])
-        if 'detail' in errdata and errdata['detail']:
-            outdict['Detail'] = str(errdata['detail'])
-
-        for k in list(outdict.keys()):
-            obuf = obuf + k + ": " + outdict[k] + "\n"
-
-    except Exception as err:
-        obuf = str(payload)
-
-    ret = obuf
-    return(ret)
-
-def doexit(ecode):
-    try:
-        sys.stdout.close()
-    except:
-        pass
-    try:
-        sys.stderr.close()
-    except:
-        pass
-    sys.exit(ecode)
-
-def make_db_params(db_connect=None, db_use_ssl=False, db_timeout=30, db_connect_timeout=120, db_pool_size=30, db_pool_max_overflow=100):
-
-    db_connect_args = {
-        'timeout': db_timeout,
-        'ssl': db_use_ssl,
-    }
-
-    db_params = {
-        'db_connect': db_connect,
-        'db_connect_args': db_connect_args,
-        'db_pool_size': db_pool_size,
-        'db_pool_max_overflow': db_pool_max_overflow,
-    }
-
-    ret = anchore_engine.db.entities.common.normalize_db_params(db_params)
-    return(ret)
-
-def connect_database(config, db_params, db_retries=1):
-    # db_connect can have secrets - remove them before logging
-    loggable_db_params = copy.deepcopy(db_params)
-    del loggable_db_params['db_connect']
-    logger.info("DB params: {}".format(json.dumps(loggable_db_params)))
-
-    rc = anchore_engine.db.entities.common.do_connect(db_params)
-    logger.info("DB connection configured: {}".format(str(rc)))
-
-    db_connected = False
-    last_db_connect_err = ""
-    for i in range(0, int(db_retries)):
-        logger.info("DB attempting to connect...")
-        try:
-            rc = anchore_engine.db.entities.common.test_connection()
-            logger.info("DB connected: {}".format(str(rc)))
-            db_connected = True
-            break
-        except Exception as err:
-            last_db_connect_err = str(err)
-            if db_retries > 1:
-                logger.warn("DB connection failed, retrying - exception: {}".format(str(last_db_connect_err)))
-                time.sleep(5)
-
-    if not db_connected:
-        raise Exception("DB connection failed - exception: " + str(last_db_connect_err))
-        
-def init_database(upgrade_module=None, localconfig=None, do_db_compatibility_check=False):
```

Comment:
ohhhh I didn't realize this was moving.... and I just drilled into the details below... feel free to ignore :(